### PR TITLE
Fix outdated and thus missleading content

### LINKED ISSUE #9: Content is missleading / outdated
> This fork aims to revive the development activity around the Space API. But we're always ready to discuss joining our forces back with the original project!

### PR DESCRIPTION
**Pull request by rnestler**
Fixes #9

### CODE REVIEW

File: content/pages/home.md [hunk 1]
```diff
@@ -21,9 +21,15 @@ The reason for the fork was that the original project at
 requests were waiting for multiple months or years and sometimes the API was
 down or the directory response was incomplete or broken.
 
-This fork aims to revive the development activity around the Space API. But
-we're always ready to discuss joining our forces back with the original
-project!
+This fork aims to revive the development activity around the Space API.
+
+As of October 2017 We are trying to work together with the original SpaceAPI
```
Comment (dbrgn) [inline content/pages/home.md]: We -> we

File: content/pages/home.md [hunk 2]
```diff
@@ -21,9 +21,15 @@ The reason for the fork was that the original project at
...
+developer. So far we set up some infrastructure and meet semi regularly to
+discuss our progress. Check out https://github.com/SpaceApi/meeting-notes for
+our discussions. We encurage anybody who wants to participate to join the
+disussion on GitHub, the #spaceapi IRC channel on freenode or our mattermost
+instance on https://chat.spaceapi.net!
```
Comment (dbrgn) [inline content/pages/home.md]: Did we already decide to promote that as "official" chat? If yes, we should also put it on the spaceapi.net website.

Comment (rnestler) [reply content/pages/home.md]: @gidsi?

Comment (gidsi) [reply content/pages/home.md]: The official chat was the irc, but we talked about letting people use the mattermost instance for the project "officially", afair.

**Comment by dbrgn** [comment]
@rnestler ping

File: content/pages/home.md (hunk 1 above)
Comment (rnestler) [reply content/pages/home.md]: Done.

**Pull request merged**